File: api-certstore/csharp/src/CertStoreException.cs
using System;

namespace tr.gov.tubitak.uekae.esya.api.infra.certstore
{
    //todo Annotation!
    //@ApiClass
    public class CertStoreException:Exception
    {
        public CertStoreException()
        { }

        public CertStoreException(String aMessage):base(aMessage)
        {
            //super(aMessage);
        }

        public CertStoreException(String aMessage, Exception aCause):base(aMessage, aCause)
        {
            //super(aMessage, aCause);
        }        
    }
}

File: api-certstore/src/tr/gov/tubitak/uekae/esya/api/infra/certstore/CertStoreException.java
package tr.gov.tubitak.uekae.esya.api.infra.certstore;



public class CertStoreException extends Exception
{
	public CertStoreException()
	{}

    public CertStoreException(String aMessage)
    {
      super(aMessage);
    }

    public CertStoreException(String aMessage, Throwable aCause)
    {
      super(aMessage, aCause);
    }

    public CertStoreException(Throwable aCause)
    {
      super(aCause);
    }
}
